Client/Frontend/Browser/Toolbars/UrlBar/TopToolbarView.swift

```diff
@@ -392,6 +392,17 @@ class TopToolbarView: UIView, ToolbarProtocol {
         }
     }
     
+    func submitLocation(_ location: String?) {
+        guard let text = location, !text.isEmpty else {
+            locationTextField?.text = location
+            return
+        }
+        
+        locationTextField?.text = text
```

Comment:
You can move it to first line in this function, 
`locationTextField?.text = text` is set here and in early return, we can simplify it and call once at the beginning of this function